docs/tsc/meetings/2019-03-11.md

```diff
@@ -0,0 +1,85 @@
+    - Mark: Happy to go away and fix up what he can.
+    - Larry: Raised that it may lead to security issues if we don’t do this. Old versions can cause problems. OpenEXR had this problem in the past.
+    - John: Mavin solved the dependency fetch for Java a while ago. We should look to solutions like this.
+    - Concerns we raised in that CMake is not the best at pulling in dependencies from the internet.
+    - Patrick: We are not just able to download binaries, but we will also need to download the source of specific versions and build locally on the machine building OCIO. We will also have to be careful about static libraries and symbols. We will have to look at name-spacing things carefully.
+    - We will need to make the TAC aware of any external code that is needed when we ship with the library.
+
+* CLA/DCO requirements
+    - Needs them signed before doing merges into the new repository.
+    - John: Resolving issues with the initial setup. Team is currently backlogged on other projects but hoping to get something done in the next couple of days.
+    - DCO when we are ready to DCO sign off with commits.
+    - Every commit will have DCO sign off.
+    - John: Some info can be pulled or referenced from the TAC guidelines.
+    - Larry: Who needs to sign off the work?
+        - John: We can refer to ​https://developercertificate.org/
+    - Michael: This isn’t required until we move to the new account. Not to worry about it for now.
+    - Want to have the Linux foundation CLA system up and running first, pre-sign all the CLAs, and do one commit to flick the switch.
+    - Larry: Have agreed on the CLA but not on all the surrounding areas e.g. signature blocks etc.
+
+* Googlegroup archiving & updated gh-pages deployment
+    - Announced we will pull down the group in a week.
+    - A script will be run to get old contend and re-import it into the new group.
+    - John: Test run will done first to see how it works.
+    - The OCIO website will be updated accordingly.
+    - The new website has been updated but is yet to be deployed.
+
+* Fixed functions in the repository
+    - Sean asked if there has been any more follow up conversations about this.
+    - Kevin thought that this wasn't a show stopper for him.
+    - Kevin raised the camera log transform and whether or not the goal is being set from the ACES requirements.
+    - May require to change the camera op to support ARRI.
+    - **TODO**: Post a review as it's been up for a couple of weeks now.
+
+* Next 1.1.x release
+    - There are a small number of commits that will need to be signed off before the release is made.
+    - Agreed to call it the 1.1.1 release.
+    - It will be the first release of the year.
+    - **TODO**: Michael to finalize things to push it forward.
+    - **TODO**: Will need to update the change log for the new version.
+
+* Items for next TSC meeting agenda:
+    - xxx
```

Comment:
If there's nothing here, we can delete this section.

Reply:
Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236